src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoEncryptedKey.cs

```diff
@@ -32,14 +32,14 @@ public EncryptedKey EncryptedKey
         public override XmlElement GetXml()
         {
             if (_encryptedKey == null)
-                throw new CryptographicException(SecurityResources.GetResourceString("Cryptography_Xml_InvalidElement"), "KeyInfoEncryptedKey");
+                throw new CryptographicException(SR.Cryptography_Xml_InvalidElement, "KeyInfoEncryptedKey");
```

Comment:
Just checking - you're happy to tie XML element names to class names? I would have thought those would do better as regular constants...